wordssite/words/models.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from django.db import models
+from django.utils import timezone
+
+class Word(models.Model):
+    word_text = models.CharField(max_length=200)
+    word_translation = models.CharField(max_length=200, default='DEFAULT VALUE')
+    pub_date = models.DateTimeField('date published')
```

Comment:
We could have smth like BaseModel, which would have 3 fields:
uuid as primary key instead of id
created_at -> creation date of a model
updated_at -> to catch each time it is updated

and then other models could inherit from this BaseModel.